website/guides/terraform-provider-development-program.html.md

```diff
@@ -223,10 +223,10 @@ in the section above.
 
 * Address all review feedback, ensure that each resource has a corresponding  acceptance test, and the documentation is complete
 
-* Create a PR for the provider against the HashiCorp provided empty repo.
-
 * Plan to continue supporting the provider with additional functionality as well as addressing any open issues.
 
+* All Terraform Providers are required to contain a MPL-2.0 open source lisence.
```

Comment:
Couple of typos, should be: All Terraform providers are required to contain an MPL-2.0 open source license. Should we add a similar note to to section 3 as well.

Reply:
I'll change the spelling and capitalization, but I think "an" should only be used before a word that starts with a vowel? 

- All Terraform providers are required to contain a MPL-2.0 open source license.